### ISSUE TITLE
用iStore OS拨号最近经常断线

### ISSUE THREAD

**Issue by Jernin** [istoreos/istoreos#1507]
反馈bug/问题模板，提建议请删除

## 1.关于你要提交的问题
为避免重复issue，请先搜索issue，确认没有类似issue再提交新issue；
注意搜索时**包括已关闭**的issue（删掉搜索框的的“is:open”条件）；

Q：是否用关键词搜索了issue? (使用 "x" 选择)
* [ ] 没有类似的issue

## 2. 详细叙述
光猫桥接拨号现在经常性断网。
### (1) 具体问题
光猫桥接拨号现在经常性断网，短则半小时长的话1小时左右。而且断线会出先拨号不成功的情况。重新安装最新系统也是不行
A：

### (2) 路由器型号和固件版本
X86软路由  ESXI8.0   固件版本
A：

### (3) 详细日志和/或截图
附重做系统以后的完整日志。
Thu Apr 18 00:03:04 2024 daemon.info pppd[18634]: No response to 5 echo-requests
Thu Apr 18 00:03:04 2024 daemon.notice pppd[18634]: Serial link appears to be disconnected.
Thu Apr 18 00:03:04 2024 daemon.info pppd[18634]: Connect time 23.3 minutes.
Thu Apr 18 00:03:04 2024 daemon.info pppd[18634]: Sent 5464945 bytes, received 41830418 bytes.
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for 106.118.2.46 on pppoe-wan.
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Leaving mDNS multicast group on interface pppoe-wan.IPv4 with address 106.118.2.46.
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Interface pppoe-wan.IPv4 no longer relevant for mDNS.
Thu Apr 18 00:03:04 2024 daemon.notice ttyd[14559]: rops_handle_POLLIN_netlink: DELADDR
Thu Apr 18 00:03:04 2024 daemon.notice ttyd[14559]: rops_handle_POLLIN_netlink: DELADDR
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Network device 'pppoe-wan' link is down
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Network alias 'pppoe-wan' link is down
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Interface 'wan_6' has link connectivity loss
Thu Apr 18 00:03:04 2024 daemon.err odhcp6c[18884]: Failed to send RS (Network unreachable)
Thu Apr 18 00:03:04 2024 daemon.err odhcp6c[18884]: Failed to send RELEASE message to ff02::1:2 (Network unreachable)
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Interface pppoe-wan.IPv6 no longer relevant for mDNS.
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Leaving mDNS multicast group on interface pppoe-wan.IPv6 with address 240e:340:9907:523d:2daf:9dff:5b95:1fff.
Thu Apr 18 00:03:04 2024 daemon.notice ttyd[14559]: rops_handle_POLLIN_netlink: DELADDR
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for 240e:340:9907:523d:2daf:9dff:5b95:1fff on pppoe-wan.
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Interface 'wan' has lost the connection
Thu Apr 18 00:03:04 2024 daemon.warn dnsmasq[1]: no servers found in /tmp/resolv.conf.d/resolv.conf.auto, will retry
Thu Apr 18 00:03:04 2024 daemon.notice netifd: wan_6 (18884): Command failed: ubus call network.interface notify_proto { "action": 0, "link-up": false, "keep": false, "interface": "wan_6" } (Permission denied)
Thu Apr 18 00:03:04 2024 daemon.err odhcp6c[18884]: Failed to send SOLICIT message to ff02::1:2 (Network unreachable)
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Interface 'wan_6' is now down
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Registering new address record for fe80::20c:29ff:fef4:1248 on br-lan.*.
Thu Apr 18 00:03:04 2024 daemon.info avahi-daemon[10702]: Registering new address record for fd09:7cda:46a::1 on br-lan.*.
Thu Apr 18 00:03:04 2024 daemon.notice netifd: Interface 'wan_6' is disabled
Thu Apr 18 00:03:05 2024 daemon.err nmbd[19786]: [2024/04/18 00:03:05.128868,  0] ../../source3/nmbd/nmbd.c:59(terminate)
Thu Apr 18 00:03:05 2024 daemon.err nmbd[19786]:   Got SIGTERM: going down...
Thu Apr 18 00:03:05 2024 daemon.err nmbd[8194]: [2024/04/18 00:03:05.151443,  0] ../../source3/nmbd/nmbd.c:901(main)
Thu Apr 18 00:03:05 2024 daemon.err nmbd[8194]:   nmbd version 4.18.8 started.
Thu Apr 18 00:03:05 2024 daemon.err nmbd[8194]:   Copyright Andrew Tridgell and the Samba Team 1992-2023
Thu Apr 18 00:03:05 2024 daemon.err smbd[8193]: [2024/04/18 00:03:05.162949,  0] ../../source3/smbd/server.c:1746(main)
Thu Apr 18 00:03:05 2024 daemon.err smbd[8193]:   smbd version 4.18.8 started.
Thu Apr 18 00:03:05 2024 daemon.err smbd[8193]:   Copyright Andrew Tridgell and the Samba Team 1992-2023
Thu Apr 18 00:03:06 2024 daemon.warn dnsmasq[1]: ignoring query from non-local network fd72:3f6d:4900:7346:d15a:cbdb:9a72:af49 (logged only once)
Thu Apr 18 00:03:10 2024 daemon.notice pppd[18634]: Connection terminated.
Thu Apr 18 00:03:10 2024 daemon.info pppd[18634]: Connect time 23.3 minutes.
Thu Apr 18 00:03:10 2024 daemon.info pppd[18634]: Sent 5464945 bytes, received 41830418 bytes.
Thu Apr 18 00:03:10 2024 daemon.debug ModemManager[8350]: hotplug: event reported: action=remove, name=pppoe-wan, subsystem=net
Thu Apr 18 00:03:10 2024 daemon.notice pppd[18634]: Modem hangup
Thu Apr 18 00:03:10 2024 daemon.info pppd[18634]: Exit.
Thu Apr 18 00:03:10 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:03:10 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:03:10 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:03:10 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:03:10 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:03:10 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:03:10 2024 daemon.info pppd[8424]: Plugin pppoe.so loaded.
Thu Apr 18 00:03:10 2024 daemon.info pppd[8424]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:03:10 2024 daemon.notice pppd[8424]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:03:25 2024 daemon.warn pppd[8424]: Timeout waiting for PADO packets
Thu Apr 18 00:03:25 2024 daemon.err pppd[8424]: Unable to complete PPPoE Discovery
Thu Apr 18 00:03:25 2024 daemon.info pppd[8424]: Exit.
Thu Apr 18 00:03:25 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:03:25 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:03:25 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:03:25 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:03:25 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:03:25 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:03:25 2024 daemon.info pppd[9448]: Plugin pppoe.so loaded.
Thu Apr 18 00:03:25 2024 daemon.info pppd[9448]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:03:25 2024 daemon.notice pppd[9448]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]: [2024/04/18 00:03:28.180344,  0] ../../source3/nmbd/nmbd_become_lmb.c:398(become_local_master_stage2)
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]:   *****
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]:
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]:   Samba name server ISTOREOS is now a local master browser for workgroup WORKGROUP on subnet 10.0.0.1
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]:
Thu Apr 18 00:03:28 2024 daemon.err nmbd[8194]:   *****
Thu Apr 18 00:03:40 2024 daemon.warn pppd[9448]: Timeout waiting for PADO packets
Thu Apr 18 00:03:40 2024 daemon.err pppd[9448]: Unable to complete PPPoE Discovery
Thu Apr 18 00:03:40 2024 daemon.info pppd[9448]: Exit.
Thu Apr 18 00:03:40 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:03:40 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:03:40 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:03:40 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:03:40 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:03:40 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:03:40 2024 daemon.info pppd[9832]: Plugin pppoe.so loaded.
Thu Apr 18 00:03:40 2024 daemon.info pppd[9832]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:03:40 2024 daemon.notice pppd[9832]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:03:55 2024 daemon.warn pppd[9832]: Timeout waiting for PADO packets
Thu Apr 18 00:03:55 2024 daemon.err pppd[9832]: Unable to complete PPPoE Discovery
Thu Apr 18 00:03:55 2024 daemon.info pppd[9832]: Exit.
Thu Apr 18 00:03:55 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:03:55 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:03:55 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:03:55 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:03:55 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:03:55 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:03:55 2024 daemon.info pppd[10215]: Plugin pppoe.so loaded.
Thu Apr 18 00:03:55 2024 daemon.info pppd[10215]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:03:55 2024 daemon.notice pppd[10215]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:04:10 2024 daemon.warn pppd[10215]: Timeout waiting for PADO packets
Thu Apr 18 00:04:10 2024 daemon.err pppd[10215]: Unable to complete PPPoE Discovery
Thu Apr 18 00:04:10 2024 daemon.info pppd[10215]: Exit.
Thu Apr 18 00:04:10 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:04:10 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:04:10 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:04:10 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:04:10 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:04:10 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:04:10 2024 daemon.info pppd[10666]: Plugin pppoe.so loaded.
Thu Apr 18 00:04:10 2024 daemon.info pppd[10666]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:04:10 2024 daemon.notice pppd[10666]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:04:25 2024 daemon.warn pppd[10666]: Timeout waiting for PADO packets
Thu Apr 18 00:04:26 2024 daemon.err pppd[10666]: Unable to complete PPPoE Discovery
Thu Apr 18 00:04:26 2024 daemon.info pppd[10666]: Exit.
Thu Apr 18 00:04:26 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:04:26 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:04:26 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:04:26 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:04:26 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:04:26 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:04:26 2024 daemon.info pppd[11058]: Plugin pppoe.so loaded.
Thu Apr 18 00:04:26 2024 daemon.info pppd[11058]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:04:26 2024 daemon.notice pppd[11058]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:04:41 2024 daemon.warn pppd[11058]: Timeout waiting for PADO packets
Thu Apr 18 00:04:41 2024 daemon.err pppd[11058]: Unable to complete PPPoE Discovery
Thu Apr 18 00:04:41 2024 daemon.info pppd[11058]: Exit.
Thu Apr 18 00:04:41 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:04:41 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:04:41 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:04:41 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:04:41 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:04:41 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:04:41 2024 daemon.info pppd[11450]: Plugin pppoe.so loaded.
Thu Apr 18 00:04:41 2024 daemon.info pppd[11450]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:04:41 2024 daemon.notice pppd[11450]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:04:56 2024 daemon.warn pppd[11450]: Timeout waiting for PADO packets
Thu Apr 18 00:04:56 2024 daemon.err pppd[11450]: Unable to complete PPPoE Discovery
Thu Apr 18 00:04:56 2024 daemon.info pppd[11450]: Exit.
Thu Apr 18 00:04:56 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:04:56 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:04:56 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:04:56 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:04:56 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:04:56 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:04:56 2024 daemon.info pppd[11845]: Plugin pppoe.so loaded.
Thu Apr 18 00:04:56 2024 daemon.info pppd[11845]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:04:56 2024 daemon.notice pppd[11845]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:05:11 2024 daemon.warn pppd[11845]: Timeout waiting for PADO packets
Thu Apr 18 00:05:11 2024 daemon.err pppd[11845]: Unable to complete PPPoE Discovery
Thu Apr 18 00:05:11 2024 daemon.info pppd[11845]: Exit.
Thu Apr 18 00:05:11 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:05:11 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:05:11 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:05:11 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:05:11 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:05:11 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:05:11 2024 daemon.info pppd[12232]: Plugin pppoe.so loaded.
Thu Apr 18 00:05:11 2024 daemon.info pppd[12232]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:05:11 2024 daemon.notice pppd[12232]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:05:26 2024 daemon.warn pppd[12232]: Timeout waiting for PADO packets
Thu Apr 18 00:05:26 2024 daemon.err pppd[12232]: Unable to complete PPPoE Discovery
Thu Apr 18 00:05:26 2024 daemon.info pppd[12232]: Exit.
Thu Apr 18 00:05:26 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:05:26 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:05:26 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:05:26 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:05:26 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:05:26 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:05:26 2024 daemon.info pppd[12619]: Plugin pppoe.so loaded.
Thu Apr 18 00:05:26 2024 daemon.info pppd[12619]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:05:26 2024 daemon.notice pppd[12619]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:05:41 2024 daemon.warn pppd[12619]: Timeout waiting for PADO packets
Thu Apr 18 00:05:41 2024 daemon.err pppd[12619]: Unable to complete PPPoE Discovery
Thu Apr 18 00:05:41 2024 daemon.info pppd[12619]: Exit.
Thu Apr 18 00:05:41 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:05:41 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:05:41 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:05:41 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:05:41 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:05:41 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:05:41 2024 daemon.info pppd[13005]: Plugin pppoe.so loaded.
Thu Apr 18 00:05:41 2024 daemon.info pppd[13005]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:05:42 2024 daemon.notice pppd[13005]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:05:57 2024 daemon.warn pppd[13005]: Timeout waiting for PADO packets
Thu Apr 18 00:05:57 2024 daemon.err pppd[13005]: Unable to complete PPPoE Discovery
Thu Apr 18 00:05:57 2024 daemon.info pppd[13005]: Exit.
Thu Apr 18 00:05:57 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:05:57 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:05:57 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:05:57 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:05:57 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:05:57 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:05:57 2024 daemon.info pppd[13393]: Plugin pppoe.so loaded.
Thu Apr 18 00:05:57 2024 daemon.info pppd[13393]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:05:57 2024 daemon.notice pppd[13393]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:06:12 2024 daemon.warn pppd[13393]: Timeout waiting for PADO packets
Thu Apr 18 00:06:12 2024 daemon.err pppd[13393]: Unable to complete PPPoE Discovery
Thu Apr 18 00:06:12 2024 daemon.info pppd[13393]: Exit.
Thu Apr 18 00:06:12 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:06:12 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:06:12 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:06:12 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:06:12 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:06:12 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:06:12 2024 daemon.info pppd[13780]: Plugin pppoe.so loaded.
Thu Apr 18 00:06:12 2024 daemon.info pppd[13780]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:06:12 2024 daemon.notice pppd[13780]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:06:27 2024 daemon.warn pppd[13780]: Timeout waiting for PADO packets
Thu Apr 18 00:06:27 2024 daemon.err pppd[13780]: Unable to complete PPPoE Discovery
Thu Apr 18 00:06:27 2024 daemon.info pppd[13780]: Exit.
Thu Apr 18 00:06:27 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:06:27 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:06:27 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:06:27 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:06:27 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:06:27 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:06:27 2024 daemon.info pppd[14165]: Plugin pppoe.so loaded.
Thu Apr 18 00:06:27 2024 daemon.info pppd[14165]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:06:27 2024 daemon.notice pppd[14165]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:06:42 2024 daemon.warn pppd[14165]: Timeout waiting for PADO packets
Thu Apr 18 00:06:42 2024 daemon.err pppd[14165]: Unable to complete PPPoE Discovery
Thu Apr 18 00:06:42 2024 daemon.info pppd[14165]: Exit.
Thu Apr 18 00:06:42 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:06:42 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:06:42 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:06:42 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:06:42 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:06:42 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:06:42 2024 daemon.info pppd[14564]: Plugin pppoe.so loaded.
Thu Apr 18 00:06:42 2024 daemon.info pppd[14564]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:06:42 2024 daemon.notice pppd[14564]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:06:57 2024 daemon.warn pppd[14564]: Timeout waiting for PADO packets
Thu Apr 18 00:06:57 2024 daemon.err pppd[14564]: Unable to complete PPPoE Discovery
Thu Apr 18 00:06:57 2024 daemon.info pppd[14564]: Exit.
Thu Apr 18 00:06:57 2024 daemon.notice netifd: Interface 'wan' is now down
Thu Apr 18 00:06:57 2024 daemon.notice netifd: Interface 'wan' is setting up now
Thu Apr 18 00:06:57 2024 daemon.err insmod: module is already loaded - slhc
Thu Apr 18 00:06:57 2024 daemon.err insmod: module is already loaded - ppp_generic
Thu Apr 18 00:06:57 2024 daemon.err insmod: module is already loaded - pppox
Thu Apr 18 00:06:57 2024 daemon.err insmod: module is already loaded - pppoe
Thu Apr 18 00:06:57 2024 daemon.info pppd[14968]: Plugin pppoe.so loaded.
Thu Apr 18 00:06:57 2024 daemon.info pppd[14968]: PPPoE plugin from pppd 2.4.9
Thu Apr 18 00:06:57 2024 daemon.notice pppd[14968]: pppd 2.4.9 started by root, uid 0
Thu Apr 18 00:07:02 2024 daemon.info pppd[14968]: PPP session is 9428
Thu Apr 18 00:07:03 2024 daemon.warn pppd[14968]: Connected to 58:f9:87:da:12:68 via interface eth0
Thu Apr 18 00:07:03 2024 kern.info kernel: [ 1727.944164] pppoe-wan: renamed from ppp0
Thu Apr 18 00:07:03 2024 daemon.debug ModemManager[15268]: hotplug: event reported: action=add, name=ppp0, subsystem=net
Thu Apr 18 00:07:03 2024 daemon.warn [10765]: <warn>  [ppp0] invalid sysfs path read for net/ppp0
Thu Apr 18 00:07:03 2024 daemon.warn [10765]: <warn>  [base-manager] couldn't handle kernel event: device net/ppp0 not found
Thu Apr 18 00:07:03 2024 daemon.err ModemManager[15268]: hotplug: parent device sysfspath not found
Thu Apr 18 00:07:03 2024 daemon.debug ModemManager[15320]: hotplug: event reported: action=move, name=pppoe-wan, subsystem=net
Thu Apr 18 00:07:03 2024 daemon.warn [10765]: <warn>  [base-manager] couldn't handle kernel event: Invalid 'action' parameter given: 'move' (expected 'add' or 'remove')
Thu Apr 18 00:07:03 2024 daemon.info pppd[14968]: Renamed interface ppp0 to pppoe-wan
Thu Apr 18 00:07:03 2024 daemon.info pppd[14968]: Using interface pppoe-wan
Thu Apr 18 00:07:03 2024 daemon.notice pppd[14968]: Connect: pppoe-wan <--> eth0
Thu Apr 18 00:07:06 2024 daemon.info pppd[14968]: Remote message: Authentication success,Welcome!
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: PAP authentication succeeded
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: peer from calling number 58:F9:87:DA:12:68 authorized
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: local  LL address fe80::8532:6b97:f9c4:e7ec
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: remote LL address fe80::5af9:87ff:feda:1268
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Joining mDNS multicast group on interface pppoe-wan.IPv6 with address fe80::8532:6b97:f9c4:e7ec.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: New relevant interface pppoe-wan.IPv6 for mDNS.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Registering new address record for fe80::8532:6b97:f9c4:e7ec on pppoe-wan.*.
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: local  IP address 106.118.3.231
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: remote IP address 106.118.0.1
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: primary   DNS address 222.222.222.222
Thu Apr 18 00:07:06 2024 daemon.notice pppd[14968]: secondary DNS address 222.222.202.202
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Joining mDNS multicast group on interface pppoe-wan.IPv4 with address 106.118.3.231.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: New relevant interface pppoe-wan.IPv4 for mDNS.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Registering new address record for 106.118.3.231 on pppoe-wan.IPv4.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for 106.118.3.231 on pppoe-wan.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Leaving mDNS multicast group on interface pppoe-wan.IPv4 with address 106.118.3.231.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Interface pppoe-wan.IPv4 no longer relevant for mDNS.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Joining mDNS multicast group on interface pppoe-wan.IPv4 with address 106.118.3.231.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: New relevant interface pppoe-wan.IPv4 for mDNS.
Thu Apr 18 00:07:06 2024 daemon.info avahi-daemon[10702]: Registering new address record for 106.118.3.231 on pppoe-wan.IPv4.
Thu Apr 18 00:07:06 2024 daemon.notice ttyd[14559]: rops_handle_POLLIN_netlink: DELADDR
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Network device 'pppoe-wan' link is up
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Network alias 'pppoe-wan' link is up
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Interface 'wan' is now up
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: reading /tmp/resolv.conf.d/resolv.conf.auto
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using nameserver 222.222.222.222#53
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using nameserver 222.222.202.202#53
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for test
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for onion
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for localhost
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for local
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for invalid
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for bind
Thu Apr 18 00:07:06 2024 daemon.info dnsmasq[1]: using only locally-known addresses for lan
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Interface 'wan_6' is enabled
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Interface 'wan_6' has link connectivity
Thu Apr 18 00:07:06 2024 daemon.notice netifd: Interface 'wan_6' is setting up now
Thu Apr 18 00:07:06 2024 user.notice firewall: Reloading firewall due to ifup of wan (pppoe-wan)
Thu Apr 18 00:07:06 2024 user.notice firewall: Reloading firewall due to ifupdate of wan (pppoe-wan)
Thu Apr 18 00:07:06 2024 user.notice ddns-scripts[15574]: Jernin: PID '15574' started at 2024-04-18 00:07
Thu Apr 18 00:07:07 2024 daemon.err nmbd[8194]: [2024/04/18 00:07:07.096660,  0] ../../source3/nmbd/nmbd.c:59(terminate)
Thu Apr 18 00:07:07 2024 daemon.err nmbd[8194]:   Got SIGTERM: going down...
Thu Apr 18 00:07:07 2024 daemon.err nmbd[16212]: [2024/04/18 00:07:07.121230,  0] ../../source3/nmbd/nmbd.c:901(main)
Thu Apr 18 00:07:07 2024 daemon.err nmbd[16212]:   nmbd version 4.18.8 started.
Thu Apr 18 00:07:07 2024 daemon.err nmbd[16212]:   Copyright Andrew Tridgell and the Samba Team 1992-2023
Thu Apr 18 00:07:07 2024 daemon.err smbd[16211]: [2024/04/18 00:07:07.127034,  0] ../../source3/smbd/server.c:1746(main)
Thu Apr 18 00:07:07 2024 daemon.err smbd[16211]:   smbd version 4.18.8 started.
Thu Apr 18 00:07:07 2024 daemon.err smbd[16211]:   Copyright Andrew Tridgell and the Samba Team 1992-2023
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Leaving mDNS multicast group on interface pppoe-wan.IPv6 with address fe80::8532:6b97:f9c4:e7ec.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Joining mDNS multicast group on interface pppoe-wan.IPv6 with address 240e:340:9907:5257:8532:6b97:f9c4:e7ec.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Registering new address record for 240e:340:9907:5257:8532:6b97:f9c4:e7ec on pppoe-wan.*.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for fe80::8532:6b97:f9c4:e7ec on pppoe-wan.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Registering new address record for 240e:342:9954:cc00::1 on br-lan.*.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for 240e:342:9954:a900::1 on br-lan.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for fe80::20c:29ff:fef4:1248 on br-lan.
Thu Apr 18 00:07:08 2024 daemon.info avahi-daemon[10702]: Withdrawing address record for fd09:7cda:46a::1 on br-lan.
Thu Apr 18 00:07:08 2024 daemon.notice netifd: Interface 'wan_6' is now up
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: reading /tmp/resolv.conf.d/resolv.conf.auto
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using nameserver 222.222.222.222#53
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using nameserver 222.222.202.202#53
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using nameserver 240e:4c:4008::1#53
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using nameserver 240e:4c:4808::1#53
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for test
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for onion
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for localhost
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for local
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for invalid
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for bind
Thu Apr 18 00:07:08 2024 daemon.info dnsmasq[1]: using only locally-known addresses for lan
Thu Apr 18 00:07:08 2024 user.notice firewall: Reloading firewall due to ifup of wan_6 (pppoe-wan)
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]: [2024/04/18 00:07:30.149463,  0] ../../source3/nmbd/nmbd_become_lmb.c:398(become_local_master_stage2)
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]:   *****
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]:
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]:   Samba name server ISTOREOS is now a local master browser for workgroup WORKGROUP on subnet 10.0.0.1
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]:
Thu Apr 18 00:07:30 2024 daemon.err nmbd[16212]:   *****
Thu Apr 18 00:17:10 2024 user.warn ddns-scripts[15574]: Jernin: Updating IP at DDNS provider failed - starting retry 1/0

**Comment by bailuxianfeiying** [comment]
有可能是光猫问题，我家的光猫桥接模式也频繁断线

**Comment by jjm2473** [comment]
之前有用户反馈也是光猫问题，找运营商换光猫就好了

**Comment by kc-bao** [comment]
此问题已在 **酷友社** 上被提及。那里可能有相关详细信息：

https://www.koolcenter.com/t/topic/2299/1
